node/full.go

```diff
@@ -286,8 +318,8 @@ func newQitmeerFullNode(node *Node) (*QitmeerFull, error) {
 	}
 	// init address api
 	qm.addressApi = address.NewAddressApi(cfg, node.Params, qm.GetBlockChain())
-
-	if err := qm.RegisterAccountService(cfg); err != nil {
+	autoCollectOp := make(chan types.AutoCollectUtxo, 1)
```

Comment:
If you want to transfer data between different modules, I suggest considering using  `Consensus.Events()` to decouple